tests/reframe/config/settings.py

```diff
@@ -0,0 +1,87 @@
+site_configuration = {
+    'systems': [
+        {
+            'name': 'example_system',
```

Comment:
this looks like cartesius ;) let's make sure we have here a default partition with a launcher that is local e.g. mpirun; so that it does not rely on a non-eessi environment (like slurm)

Reply:
Yep, I'll take it out. We have the Magic Castle settings file as an example now.